ckanext/emailasusername/i18n/fr/LC_MESSAGES/ckanext-emailasusername.po

```diff
@@ -56,19 +59,19 @@ msgstr "Nom d'utilisateur"
 
 #: ckanext/emailasusername/templates/user/new_user_form.html:5
 msgid "username"
-msgstr "nom d'utilisateur"
+msgstr "Nom d'utilisateur"
 
 #: ckanext/emailasusername/templates/user/new_user_form.html:6
 msgid "Full Name"
-msgstr "Nom Complet"
+msgstr "Nom"
 
 #: ckanext/emailasusername/templates/user/new_user_form.html:6
 msgid "Joe Bloggs"
-msgstr "Joe Bloggs"
+msgstr ""
```

Comment:
This is an example name given as a placeholder in a name field somewhere. If the translated string is empty, the place holder will be removed? Just wondering if it really should be empty?

Reply:
No, it won't be removed. In french it'll still say "Joe Bloggs".